diagnostic.md

```diff
@@ -3,36 +3,38 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+routability, less AJAX, responsiveness
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+It IS handlebars! Pretty much. It also uses routes and components, but it's basically handlebars
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+Router, template, model, component
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+a component can be reused in other areas of your application, whereas templates only are used within a particular pod
```

Comment:
Sort of. Components are not routable, and they are often re-usable parts of larger templates. 
However, regular templates can also be reused.
.5/1